exchanges/deribit/deribit_websocket.go

```diff
@@ -0,0 +1,1212 @@
+package deribit
```

Comment:
I'm having websocket issues:
```ERROR] | EXCHANGE | 15/06/2023 13:03:53 | subscription to channel chart.trades.BTC_FS-28JUL23_30JUN23.60 was not successful```

The sync manager is not subscribing or updating orderbook or ticker entries. It can only do so via REST. I'm not doing anything fancy like enabling all pairs either.